kolibri/plugins/coach/assets/src/views/reports/channel-list-page.vue

```diff
@@ -46,6 +42,12 @@
         </tr>
       </tbody>
     </core-table>
+
+    <template v-if="!standardDataTable.length">
```

Comment:
Thought on adding `v-if="standardDataTable.length"` to the tbody so we don't just have an empty with an empty tbody?

Reply:
I take that bad. Don't think it is semantically correct to omit the tbody.